packages/patternfly-4/react-core/src/components/Dropdown/DropdownIcon.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/react-styles/css/components/Dropdown/dropdown';
+
+export interface DropdownIconProps extends React.HTMLProps<HTMLAnchorElement> {
+  /** Icon to be rendered in the dropdown item */
+  children?: React.ReactNode;
+  /** Classes applied to span element of dropdown icon item */
+  className?: string;
+}
+
+export const DropdownIcon: React.FunctionComponent<DropdownIconProps> = ({
```

Comment:
Should we call this `DropdownItemIcon`? That would align closer with what's in core and reserve `DropdownIcon` for some other thing we might add down the road.